typescript/web/src/connectors/resolvers/workspace.ts

```diff
@@ -19,8 +23,10 @@ const workspaces = async (
   { repository }: Context
 ) => await repository.workspace.list(args.where);
 
-const datasets = (_parent: any, _args: any, { repository }: Context) => {
-  return repository.dataset.list();
+const datasets = async (_parent: any, _args: any, { repository }: Context) => {
```

Comment:
I known you're not the author but It's cleaner to use `never` instead of `any` if you don't use these args.

```suggestion
const datasets = async (_parent: never, _args: never, { repository }: Context) => {
```